apps/mobile/src/screens/Dashboard/Home/Widget.tsx

```diff
@@ -1,13 +1,29 @@
-import { useSnapshot, WidgetFeature } from '@walless/app';
+import type { ViewStyle } from 'react-native';
+import {
+	useSafeAreaInsets,
+	useSnapshot,
+	WidgetExplorerFeature,
+	WidgetFeature,
+} from '@walless/app';
 import { appState } from '@walless/engine';
 
-import ExplorerScreen from '../Explore';
-
 export const WidgetScreen = () => {
+	const insets = useSafeAreaInsets();
 	const { activeWidgetId } = useSnapshot(appState);
+	const explorerContainerStyle: ViewStyle = {
+		paddingTop: insets.top,
+	};
+	const explorerScrollStyle: ViewStyle = {
+		paddingBottom: insets.bottom + insets.bottom,
```

Comment:
why there is a `insets.bottom + insets.bottom`?

Reply:
oops, nice catch lol